coilsnake/modules/eb/ExpandedTablesModule.py

```diff
@@ -2,8 +2,7 @@
 from coilsnake.model.eb.table import eb_table_from_offset
 from coilsnake.modules.eb.EbModule import EbModule
 from coilsnake.util.common.yml import yml_load
-from coilsnake.util.eb.pointer import write_asm_pointer, from_snes_address, to_snes_address
-
+from coilsnake.util.eb.pointer import write_asm_pointer, write_xl_pointer, from_snes_address, to_snes_address
 
 class AsmPointer(object):
```

Comment:
Can you:
1. Rename `AsmPointer` to `AsmPointerReference` (and similar with `XlPointer`)
2. Rename `write` to `relocate`
3. Move both classes to `coilsnake/util/eb/pointer.py`
4. Remove the subclassing of `object` (this is a Python2 leftover, and isn't necessary anymore)

Thanks!
P.S. If you have time and want to, go ahead and add a base class `PointerReference` with a `relocate` method which just raises `NotImplementedError` :)